src/neo4j_genai/embeddings/sentence_transformers.py

```diff
@@ -1,8 +1,11 @@
 from neo4j_genai.embedder import Embedder
+from typing import Any
 
 
 class SentenceTransformerEmbeddings(Embedder):
-    def __init__(self, model="all-MiniLM-L6-v2", *args, **kwargs):
+    def __init__(
+        self, model: str = "all-MiniLM-L6-v2", *args: Any, **kwargs: Any
```

Comment:
Wow, that's a pretty ugly syntax :( Do you know the reason why we need to add types to args and kwargs? Do we have the ability to add constraints like 'any kwarg must be a float'?

Reply:
https://typing.readthedocs.io/en/latest/spec/callables.html#unpack-kwargs Based on this it is possible, are we able to restrict the type of these further?

Reply:
🤯

No I don't think we can restrict it, but I was not aware of this feature, nice :)